lib/vintage_net_wizard/web/router.ex

```diff
@@ -60,6 +60,30 @@ defmodule VintageNetWizard.Web.Router do
     render_password_page(conn, key_mgmt, ssid: ssid, password: "", error: "", user: "")
   end
 
+  get "/redirect" do
+    redirect(conn, "/")
+  end
+
+  get "/ncsi.txt" do
+    redirect(conn, "/")
+  end
+
+  get "/connecttest.txt" do
+    redirect(conn, "/")
+  end
+
+  get "/generate_204" do
+    redirect(conn, "/")
+  end
+
+  get "/hotspot-detect.html" do
+    render_page(conn, "apple_captive_portal.html")
+  end
+
+  get "/library/test/success.html" do
+    render_page(conn, "apple_captive_portal.html")
+  end
+
```

Comment:
Are there docs for all the things that need to redirect or even the special apple case? Just wondering if there's a doc that we can cross-reference to know that we have everything.

Another option would be to find a big OSS project that we can periodically steal from as magic paths get added.

Reply:
I ended up doing lots of googling to find the addresses, I've implemented a captive portal in the past so could remember some of them.

I've been trying to find a decent source but it appears to be hard, I did find this repo that list all of them: https://github.com/Embedded-Chiangdi/GB_Code/blob/master/Wifidog_Note/portal.md

Reply:
This is helpful. Thanks.